src/wasm-binary.h

```diff
@@ -155,7 +155,8 @@ enum Section {
   Functions = 2,
   DataSegments = 4,
   FunctionTable = 5,
-  End = 6
+  End = 6,
+  WLLSegment = 0x11
```

Comment:
Because it is not something more 'generic' it is the section reserved for the WLL project!